### ISSUE TITLE
devenv.exe, An unexpected error occurred

### ISSUE THREAD

**Issue by SplatRT** [microsoft/PTVS#5368]
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 16.1.19129.1

System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.VisualStudio.Text.Tagging.SimpleTagger`1.UpdateBatchSpan(ITrackingSpan snapshotSpan)
   at Microsoft.VisualStudio.Text.Tagging.SimpleTagger`1.<>c__DisplayClass11_0.<RemoveTagSpans>b__0(TrackingTagSpan`1 tagSpan)
   at System.Collections.Generic.List`1.RemoveAll(Predicate`1 match)
   at Microsoft.VisualStudio.Text.Tagging.SimpleTagger`1.RemoveTagSpans(Predicate`1 match)
   at Microsoft.PythonTools.Intellisense.TaskProvider.<RefreshAsync>d__28.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.PythonTools.Intellisense.TaskProvider.Refresh()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

**Comment by RaymonGulati1** [comment]
Thank you for reporting this issue. Can you please provide some more information about it?

What version of VS are you using? 2017? 2019?
What are the steps to reproduce it?

**Comment by RaymonGulati1** [comment]
@SplatRT, I'd like to look into this issue, but I need more information. Can you please tell me how to reproduce it?

**Comment by SplatRT** [comment]
I don't know how to reproduce it.

I am relatively new to python so had a syntax error that I was trying to fix.
I was trying all sorts of things to try to get it to work until this happened:
![Screenshot](https://user-images.githubusercontent.com/39800557/60062918-076b2000-9736-11e9-9c7f-aeae015b1177.png)

(I vaguely remember holding the _CTRL_ key at the time the error occurred.)

**Comment by huguesv** [comment]
Created internal bug 941517

**Comment by huguesv** [comment]
Fixed for 16.3 Preview 2.
